build-tests/rush-redis-cobuild-plugin-integration-test/sandbox/sharded-repo/projects/e/config/rush-project.json

```diff
@@ -1,10 +1,16 @@
 {
+  "$schema": "../../../../../../../libraries/rush-lib/src/schemas/rush-project.schema.json",
   "operationSettings": [
     {
       "operationName": "_phase:build",
       "outputFolderNames": ["dist"],
+      "allowCobuildWithoutCache": true,
+      "disableBuildCacheForOperation": true,
       "sharding": {
-        "count": 75
+        "count": 75,
+        "shardOperationSettings": {
+          "allowCobuildWithoutCache": true
```

Comment:
What is the difference between this and the setting on line 7?

Reply:
One applies to the shard operation itself and the other to the collate operation from #4652.

Reply:
Can you name them more specifically? Calling them the same thing makes one assume that the one higher up in the file is the same thing, but applies more broadly.

Reply:
How about moving this up to operation settings as a separate property, something like 
```
{
  "operationSettings": [
    {
      "operationName": "_phase:build",
      "outputFolderNames": ["registry-artifact"]
    },
   {
     "operationName": "_phase:build:shard",
      "outputFolderNames": ["another-folder"]
    }
  ]
}
```

Reply:
@iclanton gentle bump on my comment above - if that makes sense, would you be open to having that be a followup PR?

Reply:
I think that makes more sense.

Reply:
addressed in https://github.com/microsoft/rushstack/pull/4923